SearchActivity.py

```diff
@@ -37,12 +38,18 @@
 from json import dump as jdump
 from io import StringIO
 
+try:
+    from sugar3.presence.wrapper import CollabWrapper
+except (ImportError, ModuleNotFoundError):
+    from textchannelwrapper import CollabWrapper
```

Comment:
`CollabWrapper` is maintained independently [here](https://github.com/sugarlabs/collabwrapper) and isn't part of the toolkit, the documentation was to show how to use it.

You can just use `collabwrapper.py` instead of `textchannelwrapper`.